Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -298,8 +302,9 @@ public class ContainerStackView: UIView {
         }
     }
 
-    func showSubview(_ subview: UIView, animated: Bool = true) {
-        assert(subviews.contains(subview))
+    public func showSubview(_ subview: UIView, animated: Bool = true) {
+        guard subviews.contains(subview) else { return }
+        guard isHidden(subview) else { return }
```

Comment:
tbh: I'd find doing `guard subview.alpha == 0 else { return }` less confusing than this helper.